src/main/scala/nl/knaw/dans/api/sword2/CollectionDepositManagerImpl.scala

```diff
@@ -43,12 +43,12 @@ class CollectionDepositManagerImpl extends CollectionDepositManager {
     if(iri != SwordProps("collection.iri")) throw new SwordError("http://purl.org/net/sword/error/MethodNotAllowed", 405, s"Not a valid collection IRI: $iri")
   }
 
-  private def setDepositStateToDraft(id: String, userId: String): Try[Unit] = Try {
+  private def setDepositStateToDraft(id: String, userId: String): Try[Unit] =
```

Comment:
Called method already returns a `Try[Unit]`, wrapping it again in a `Try` will make `Failure`s invisible to the client.